src/overview/components/Overview.jsx

```diff
@@ -21,16 +21,17 @@ const Overview = props => (
         <Head />
         <Header {...props} />
 
-        <Tooltip
+        {/* <Tooltip
             showTooltip={props.showTooltip}
             toggleShowTooltip={props.toggleShowTooltip}
             tooltip={props.tooltip}
             fetchNextTooltip={props.fetchNextTooltip}
             isTooltipRenderable={props.isTooltipRenderable}
-        />
+        /> */}
+        {props.sidebarIcons}
+        <SideBar />
 
         <ShareButtons />
-
         <div className={showFilterClass(props)}>{props.filters}</div>
```

Comment:
I would also ensure this line is removed, along with the connected states and where `props.filters` is passed down to this component, etc..